python/mxnet/ndarray/numpy/random.py

```diff
@@ -515,6 +515,92 @@ def weibull(a, size):
         return _npi.weibull(a=a, size=size)
 
 
+def pareto(a, size):
```

Comment:
We need to follow the NumPy interface exactly. Please consider changing this to
```python
pareto(a, size=None)
```

Reply:
Done.